MyWebPage/index.html

```diff
@@ -33,6 +33,39 @@ <h2>Play Billar</h2>
c2VaZJkxG9qT+Grd2i2XinMGcSHhISHYgbyYBiTYWvEelcVTmSQSNbj+tOne1+LUx+zl1XdR8NtBwnl0owENhxnWQDI0mJ8yeaGaINiSRoSInlv+++VBi+0KiqH8Ow9BsSnKseDhzQfACthtvDHdUvENiwyvhOJa9F5lDyKarryyZNC5goHlxtWPX8Pw78hEZQbdAZb6LneG0HRYiNPwZHkB5K1r2Qw/8AAGHDr/7R0Nrj/wDC5mSf4k0oxmwUoWAHu6UcxAxKS0bcUrEtmJ4KpIrCDVJI96Z4PacpLOKKlsqNlXUttWgWibxzR/YrfBxNES2oXDQiT0kmY0GzO4qBQxNESyoXDQiTHUmY0GyTuMwp9qIU2xCoCnVz8QV+7QLXSSDeB6Uq2TgS88luYButX5UC61enuRRXaAZS0jMFJQwgAp+EySSoeNHbNCsOg5EZ3lxmsTCdQ1bS+8vyTxmpFRwo7Q+pxMbhzvkAI3q4qObQ2m/U8mNwHHhAjrZFbV7RPofy4ZzIhoBsN2jcMfArjond/LTXZ3+JjyCE4hmf9IKT/Cv/APqkg7PhaZ7t1KuKW3Gno8UEoUJ/1KNqEc2I4iwcbuYCHJZUf/1vBIV5TQms8PqNFMxItp12gY/u4EIDG4EtDLAi05dS4Eg+Z059T2d24weIGUuBubZXCE68N7dPkTVr2FsPMkhtyEm4IvF+E+39K+f/APhziVBLrK0SYnKUfdJ8hXYOyu03sEhDYaKk5JVY/GreI5CJA1rQwnhVIGWfMCCQDBFuOXACDmgPwtJlRnzFzc4sbDiOJEhNe1OziyQVZVTx0PC5jjrVfZxpSdxZHjceov8AOte1facPuQbRaOI5+81Xe/GoP0pmrTjIwY3d37sm2DSY3cldGtqzZ5vOnmkAn2+oqJ3AYJ0yiUnmN0j0+tVNvGKGivWjEbV/7iJ6jX70uKjwdUYOcBCsoRjWU/ucSXEaZXYX6K19Iqs4oSqcQz42kUdhNpCf3bsHkvX11pqjap/6rYUOYj5j7VcVKbrGxVxXcM0sTsLBYkpSw73SiNDvAn/Soz5CK0d7LYzBgqSEOI1JQrKf4Fx7GnZwOCf/AAwT5e4+ta4vYWIaQUs4hzIdUKOZPlqB5AU4MVWsAZGmY9Z9PRWDabhJHVVtntHlMLEHkZT87H1p7s/bKFcYPI2qvPYR1B328w/XO1MEsYJ7IkA4dZsSCEgnw0J8ZojK7XD52xxB9gZ9wqOpXhp774KxDEjnW6cTNKMR2VxjQll1t9HALlpXkbpPtSdW3e6VkfStpQ5wofxJJEVYGm76Xjrb3t6qppPGY8r/AK+iuaXq97yqujtI1OpI/MN4e1MG9tYcie9T6x7VZ1Go3MFDDhquUPgDKBrXjqRllUTzrct3znlFDhGY30F/sPSsYJxbNYeRqZi2leBFsugi8+1bFcEVs6njoTxvXLkCl6DRraQRe9RLRzM3NvlevUKJmNBqalcVKFAWHGvCmxA16W0qBtdir0qZkCL+vWuXFbgGRewFRvqJUMuvWvVK3oHDnWqQNdSOPn8qlQplqsdOtbtqOWTHh0qMN3JNahWawMVZri3JREohxIVqYjXr0oVY4cPpUtiIFuZ8OdSlQCSfOtAPaRdDyS83N9AK9cbsJ43Bi1qOYSCDBuYmRp/SpHrgiRwT9wK51EOErg8gwkzrII1g8xWqMMCUhRtx+vtTHH4cfhHkBrQGUz+tTSVRjmGEdj5ErdxdwUi4NtDE8elRvvuFO8sxxAMDziJrZ0ERItW6QONCgTJAUlrSZICBQ1lIIkHgRb0irZ2QOMxLww7TmcqkAPStNhJmbwADxpGlubxx/Qo7ZG0F4d0OsqKFiYUDcZgR7gkUVopvI+I0HmJ76QhVmNqCHCe9ytOynSzixh3mg0pKyknDrW22YN5aG6oW4128Yhj9nzSkoy5uFfMr+NX3hdzHPMlU8Z1ph/6jxKklBVaeQE8bwBRWsotkNsJnf9z5XskqWFNIktAuIMyUT2lALqygGJ4GaUJW4nT0rF48n4rHrpWwck20iq1qm04lNMbstAKlRtTmPGKNZxqToaWkJ/EBfjWncJtBg6zSzgFeAnwUDqAanYdWn4HFJ6G4qthToMzI/XCiMPtTTMPMVW8Xuqluisze11D/AJiAf8ybf0ppgtuJ/C4pJ5G3uLVVGMekmAfpROVKv6VH05SFUtV6b2rI3wlQ56fzC1bHD4Zz4hlnpH8w+tUVsqT8Cj52ovD7WWg74Mcx+oNG+K/+YSoG03Iq1f8ABi3fCvuN8YSd30G6fNJqsbc2JiVLK1ELJ1Ok/wANvYU8wWIS4JbUCeIG4r00NGd9wUB/ulJ+x9KkVKbt8d970RteM+++QXPNn4BtDwOJbcbSdVJmP4kGKYYjZaCqWMTLfDOBm/8AEz4zVzWw2bnd8bfzCxoU9l2l73dIVPHIhU+cXp2jWNBpDd+hI9jfvPdztmqZJ9vxn1K5Y0NzLbSKiKIETPOPWsrKzURapkQeJGnKpMUuUjUweH1r2srly2aOZBTpbhUCyEIsNSB7VlZUrgo1SR9qKHAqHqK9rKlcVGqeW8o+gqVu0jlWVlcoWmgF7zc/atWm4GbjXlZXLisAkwJ59D0qREfi11tXtZVgTK5S96SZ0tXsiU8xJr2sp6m8y5BcAtlH8fhHUnpUDmHKiSFfDw5/2rKyjPptfYqu1s3CwtJKRPA+tbBEgSIHKvKys6o0T0RwtyeCY86j7gZteFZWUEKVsnDJiJN9PDjWdxrJED5VlZUklVlRKw6lREAfevCwRc5hGtZWV0qdpaBwiBw8amZdB0knwrKypKsQp0KEGPapWx0nnasrKqqleHDINhY1KwkjRRvzvWVlRKqUZh3zEH+9ToeFZWVBF1BW0CZEpPAgxTXCbbeTZYDqesBX9fOsrK4APcGuVXBN8JtBlyyVFCvym3sdfI0V3R/yfL2rKyhPJpuhpQyF/9k=" alt="rocket-league">
```

Comment:
Wtf?, The url images are too long.
Try using shor links like this.
src="https://rocketleague.media.zestyio.com/rl_gen_blog_1920x1080.f1cb27a519bdb5b6ed34049a5b86e317.jpg"